app/controllers/results_controller.rb

```diff
@@ -688,10 +693,10 @@ def view_marks
     if @result.is_a_review?
       if @current_user.is_reviewer_for?(@assignment.pr_assignment, @result) ||
           !@grouping.membership_status(current_user).nil? || !@current_user.student?
-        @mark_criteria = @assignment.get_criteria(:peer)
+        @mark_criteria = @assignment.criteria.where(&:peer_visible)
```

Comment:
Create an association `peer_criteria` (analogous to `ta_criteria`) and use it here.